### DISCUSSION TITLE
Placeholder in FluentSelect

### DISCUSSION THREAD

**Discussion by MartiusBr** [microsoft/fluentui-blazor#1027] [Ideas]
Hi guys I'm new using this UI Framework and I really like it. Would you consider adding a placeholder text to the FluentSelect component, so that it's displayed before the user selects one of the options, similar to the Placeholder parameter in the FluentCombobox component? Thank you for your attention

**Comment by dvoituron** [comment]
Hi,
The **FluentSelect** component cannot display a placeholder, as **one element is automatically selected** (the first if not defined).
In this example, the element "One" will be selected. That's the same with the HTML **select** element (even though, in HTML, it would be possible to add a `disabled selected hidden` element that FluentSelect doesn't support)

```xml
<FluentSelect TOption="string">
    <FluentOption>One</FluentOption>
    <FluentOption>Two</FluentOption>
    <FluentOption>Three</FluentOption>
</FluentSelect>
```

A solution could be:
1. To use a `<FluentOption Selected>Choose an item</FluentOption>` element (but always visible or removed by code after the first selection: `@bind-SelectedOption:after`)
2. To use a `FluentCombobox`
3. To use a `FluentAutocomplete`
